core/array/pack/j_spec.rb

```diff
@@ -0,0 +1,166 @@
+require File.expand_path('../../../../spec_helper', __FILE__)
+require File.expand_path('../../fixtures/classes', __FILE__)
+require File.expand_path('../shared/basic', __FILE__)
+require File.expand_path('../shared/numeric_basic', __FILE__)
+require File.expand_path('../shared/integer', __FILE__)
+
+ruby_version_is '2.3' do
+
+  platform_is wordsize: 64 do
+
+    describe "Array#pack with format 'J'" do
+      it_behaves_like :array_pack_basic, 'J'
+      it_behaves_like :array_pack_basic_non_float, 'J'
+      it_behaves_like :array_pack_arguments, 'J'
+      it_behaves_like :array_pack_numeric_basic, 'J'
+      it_behaves_like :array_pack_integer, 'J'
+    end
+
+    describe "Array#pack with format 'j'" do
+      it_behaves_like :array_pack_basic, 'j'
+      it_behaves_like :array_pack_basic_non_float, 'j'
+      it_behaves_like :array_pack_arguments, 'j'
+      it_behaves_like :array_pack_numeric_basic, 'j'
+      it_behaves_like :array_pack_integer, 'j'
+    end
+
+    platform_is_not :mingw32 do
+
+      describe "Array#pack with modifier '_'" do
```

Comment:
Could you nest the `describe` calls like in https://github.com/ruby/spec/blob/master/core/array/pack/l_spec.rb#L23-L24 ?
So the full description for those specs still mention 'J' :smile: